### DISCUSSION TITLE
Semicolon instead of colon

### DISCUSSION THREAD

**Discussion by derJerg** [audiamus/AaxAudioConverter#178] [Q&A] [answered]
When I convert a Book including the title of the series the outcoming directory is named: "series title #; booktitle" instead of "series title #: booktitle".
Also if i try to change title-information before converting, I just cannot enter a colon manual.

**Comment by derJerg** [comment]
Also before when I converted Books of a series by default the outcome was like Author\[#]Booktitle\files which was exactly what I wanted. What could have happened for this to change?

**Comment by audiamus** [comment]
>  I just cannot enter a colon manual.

A colon is an invalid character for directory and file names in Windows. Therefore AAX Audio Converter replaces it with a semicolon. Meta data title tags, however, will retain the colon.

> Also before when I converted Books of a series by default the outcome was like Author[#]Booktitle\files which was exactly what I wanted. What could have happened for this to change?

It's depending on conversion mode and folder structure settings. See  chapters "Output structure" (page 10) and  "Folder structure" (page 26) in the manual.

_I move this topic to the discussion section._

**Reply by derJerg** [reply]
Ok, this totally makes sense.
I found the options, but it didn't help. The option "series nuber in folder structure" is ignored.

Trying to convert a book with previewed data: author"Volker Kutscher" booktitle"Transatlantik: Gereon Rath 9" results in "Volker Kutscher\Gereon Rath 9; Transatlantik" (doesn't matter, if option is on or off)
I ripped the same book (not from the same file, I downloaded it again) a few weeks ago and the result was "Volker Kutscher\Gereon Rath\[09] Transatlantik" - just perfect.
Also "Harry Potter und der Feuerkelch - Gesprochen von Rufus Beck: Harry Potter 4" is not even recognized as part of a series as the result is "J.K.Rowling\Harry Potter und der Feuerkelch". (Well, this is possibly because she is such a bad person as the internet found out recently)

Could it be, that the meta-information for the sequenz number is not included in the file?
First I thought, there is an algorithm extracting it from the title. This would also explain, why HP didn't work, but it worked before. Also BookLibConnect recognized it as "Harry Potter - Gesprochen von Rufus Beck [4]" so the information IS existing.

What is wrong here?

P.S. I needed the manual to find all these beatiful options I was looking for so long. Why didn't you spend a button in the UI?
Anyway, thank you for your great work, I really apreciate it. Isn't there a way to donate, or did i also miss the button;) ?

**Reply by audiamus** [reply, marked as answer]
> I found the options, but it didn't help. The option "series nuber in folder structure" is ignored.

How do you download your books? If with [Book Lib Connect](https://github.com/audiamus/BookLibConnect), all series information is taken from external Audible meta data. The Book Lib Connect library window will show it. During export to .aax, one or two separate json files are created for that, which AAX Audio Converter will look for (compatibility with the defunct Audible App for Win10).  But the Audible editor may have made mistakes and series data can be incorrect. 

If you download with legacy Audible Manager or via web browser, proper series information is not available. The title meta tag is all we have. AAX Audio Converter tries to isolate the book title in it, mostly ignoring series info in that tag.   

> P.S. I needed the manual to find all these beatiful options I was looking for so long. Why didn't you spend a button in the UI?

There is one, in the system menu, top left in the header bar. This is a classic Windows dialog application, without a menu bar, adhering to the original Windows style guide. And there is the shortcut key [F1], also classic Windows.

When you start AAX Audio Converter the first time, there will be an extra popup box (it's not a nag screen!), pointing to the system menu add-ons and to context menu entries.

**Reply by derJerg** [reply]
There we have it!
I removed all of the not-aax files:) 
Thank you for helping

**Reply by audiamus** [reply]
Yes, the json files carry valuable meta information. It's clear text but it's compacted. You need a text editor with json syntax support like Visual Studio Code to make them legible. The json files are linked to the aax file through the ASIN, with a match by file name or by content. 

The Book Lib Connect manual - short as it is - has a brief intro to them on page 8: Export Settings.
Again, the manual is accessible via the system menu or [F1].
